core/Pos/Crypto/SecretSharing.hs

```diff
@@ -90,12 +90,12 @@ newtype Secret = Secret
     { getSecret :: Pvss.Secret
     } deriving (Show, Eq)
 
--- | Shares can be used to reconstruct Secret.
+-- | Shares can be used to reconstruct `Secret`.
```

Comment:
https://www.haskell.org/haddock/doc/html/ch03s08.html
It says that backtick is the same as `'` for backward compatibility. But i'll replace it, ok.